docs/config.md

```diff
@@ -0,0 +1,163 @@
+# Configuration instruction
+
+You can see the default configuration [here](../devtools/config/docker_compose_config.toml).
+
+## Basic configuration
+
+### `center_id`
+
+The center id of the server. This is used to generate the distributed unique ID through snowflake algorithm.
+
+type: `u16`
+
+### `machine_id`
+
+The machine id of the server. This is used to generate the distributed unique ID through snowflake algorithm.
+
+type: `u16`
+
+### `indexer_mode`
+
+If mercury is in the indexer mode, mercury will pull blocks from the CKB node and append the block data to the database. Otherwise, mercury will only handle RPC requests and do not append the block data.
+
+type: `bool`
+
+### `need_sync`
```

Comment:
I suggest using `allow_parallel_sync` instead of `need_sync`.

Reply:
This is a good idea.